js/app/restaurant/list.js

```diff
@@ -1,15 +1,19 @@
-import React from 'react'
+import React, {useState, useRef, useEffect} from 'react'
 import { render } from 'react-dom'
 import moment from 'moment'
 import Swiper, { Navigation } from 'swiper'
 
 import { asText } from '../components/ShippingTimeRange'
+import { useIntersection } from '../hooks/useIntersection'
 
 import 'swiper/css';
 import 'swiper/css/navigation'
 
 import './list.scss'
 
+// turn off automatic browser handle of scroll
+window.history.scrollRestoration = 'manual'
```

Comment:
This is super important. Maybe add a more explicit comment to explain even more why we do this?